### ISSUE TITLE
The interface twitches when opening a photo through a third-party explorer

### ISSUE THREAD

**Issue by Pushkin31** [SimpleMobileTools/Simple-Gallery#2452]
when opening any photo through a third-party launcher, the interface twitches.  which conductor is not important (mixplorer, solid) android 11
<details>
<summary>video</summary>

https://user-images.githubusercontent.com/35974828/167102835-889629b7-9419-42db-a901-aabe014321ef.mp4

</details>

**Comment by tibbi** [comment]
what file is that? Isnt the file or its parent hidden? Can you swipe to other images from the opened one in the gallery itself?

**Comment by Pushkin31** [comment]
no, the directory is not reserved, access is normal. I specifically created a folder "test" to demonstrate the problem (I do not want to show my personal photos) You can scroll through the files on the sides
https://user-images.githubusercontent.com/35974828/167151456-50834cdb-eff3-4eed-90e7-f29d61fb4fa9.mp4

**Comment by Denio-7** [comment]
Confirmed, same thing on Android 10. 6.22.3 works correctly.

**Comment by tibbi** [comment]
should be fixed in 6.23.6

**Comment by Pushkin31** [comment]
I'm sorry, neither 6.23.6 nor 6.23.7 ​​fixed the issue...

**Comment by Pushkin31** [comment]
I'm sorry, neither 6.23.6 nor 6.23.7 ​​fixed the issue...

**Comment by ColouringPriest** [comment]
I'd like to add that renaming a photo or multiple photos via a third-party file manager and then opening them causes the Gallery to show the first picture (when sorted to Name or Path) in the folder (Downloads in this case). Opening the Gallery app, finding the folder, and refreshing it manually fixes the issue and opens the correct picture, but that's a tedious thing to do when it used to work flawlessly a version before. On version 6.23.8 Pro from the Play Store. Android 10.

**Comment by Sergei-Alekseev** [comment]
Please fix it. 
Startup animation from file manager looks WRONG.
Tested on various devices with Android 5 - 8.

Old builds (21 for exam) works correctly but do not overwrite files when moving/copying, creating new files with (number) in filename.

**Comment by Rivan2** [comment]
It's probably related - when i open an image from an explorer it as if opens two separate windows on top of each other to show the image, that looks like a double twitch which is annoying and images eventually show up later than before. 
It started happening at 6.23.0 if i remember correctly
Also even without this glitch the version 6.22.* loads images a bit slower than older versions. The version that's quick and working properly for me is 6.21.0. I didn't test every single version after that, but 6.21.7 already seem to have an image loading delay.

Tested on Android 5

**Comment by Pushkin31** [comment]
Hello. The problem was never fixed...
